src/main/java/org/snomed/snowstorm/core/data/services/classification/RemoteClassificationServiceClient.java

```diff
@@ -1,48 +1,75 @@
 package org.snomed.snowstorm.core.data.services.classification;
 
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.activemq.command.ActiveMQTextMessage;
 import org.apache.tomcat.util.http.fileupload.util.Streams;
 import org.elasticsearch.common.Strings;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.snomed.snowstorm.core.data.services.classification.pojo.ClassificationStatusResponse;
-import org.snomed.snowstorm.core.util.TimerUtil;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.core.io.FileSystemResource;
 import org.springframework.http.*;
+import org.springframework.jms.annotation.JmsListener;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 import org.springframework.util.LinkedMultiValueMap;
 import org.springframework.util.MultiValueMap;
 import org.springframework.web.client.ResponseExtractor;
 import org.springframework.web.client.RestClientException;
 import org.springframework.web.client.RestTemplate;
 
+import javax.jms.JMSException;
 import java.io.*;
 import java.nio.file.Files;
 import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
 
 @Service
 class RemoteClassificationServiceClient {
 
 	private final RestTemplate restTemplate;
-
 	private final Logger logger = LoggerFactory.getLogger(getClass());
+	private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
This will map `ActiveMQTextMessage` into our desired `ClassificationStatusResponse`.